README.md

```diff
@@ -1,7 +1,7 @@
 # shibboleth-plugin
 
 ### Min. shibboleth-idp version:
-4.3.x
```

Comment:
We should also state here that the main branch is idp 5 compatible. The version for 4 will be somewhere (insert instructions)